app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -23,12 +40,12 @@ class BreadCrumbComponent extends React.Component {
           </Breadcrumb>
           : (location.pathname.includes('/patient/') && location.pathname.includes('/visit/')) ?
             <Breadcrumb>
-              <BreadcrumbItem id="visited" onClick={router.goBack}> <a id="home"
+              <BreadcrumbItem data-id="search" onClick={this.handleRoute.bind(this)}> <a id="home"
```

Comment:
context binding for event handlers or any other function should happen in the constructor and not inside the render function. read through the following link:
https://reactjs.org/docs/handling-events.html

Reply:
Thanks @brayoh this has been fixed